libs/api/domains/education/src/lib/graphql/frigg/address.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class AddressModel {
+  @Field({ nullable: true })
+  street?: string
+
+  @Field({ nullable: true })
+  city?: string
+
+  @Field({ nullable: true })
+  postalCode?: string
+
+  @Field({ nullable: true })
+  country?: string
+
+  @Field({ nullable: true })
+  municipality?: string
+}
```

Comment:
Is this right? 🤔  `postalCode` is misspelled (should be `postCode`) and is missing `id` and `address`
```suggestion
export class AddressModel {
  @Field({ nullable: true })
  id?: string

  @Field({ nullable: true })
  address?: string

  @Field({ nullable: true })
  municipality?: string

  @Field({ nullable: true })
  postCode?: string

  @Field({ nullable: true })
  country?: string
}
```

Reply:
Fixed